src/modules/LibraryCard/components/Login.tsx

```diff
@@ -96,7 +96,11 @@ export const Login = ({saveDetails}: IProps) => {
         {!isLoading ? (
           <View style={styles.loginForm}>
             <Text style={styles.loginTitle}>{locales.loginTitle.fi}</Text>
-            <Text style={styles.errorMessage}>{errorMessage}</Text>
+            <Text
+              style={styles.errorMessage}
+               accessibilityLabel={errorMessage}
+               accessibilityRole={'text'}
```

Comment:
Should the accessibilityRole be 'alert' instead of 'text'?

Reply:
Not really. In mobile world Alert is an pop up view with alert message, this is just an error message without the alert.